Quadriga/src/main/java/edu/asu/spring/quadriga/rest/SearchNetworkRestController.java

```diff
@@ -0,0 +1,128 @@
+package edu.asu.spring.quadriga.rest;
+
+import java.io.StringWriter;
+import java.security.Principal;
+import java.util.ArrayList;
+import java.util.List;
+
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+import org.apache.velocity.Template;
+import org.apache.velocity.VelocityContext;
+import org.apache.velocity.app.VelocityEngine;
+import org.apache.velocity.exception.MethodInvocationException;
+import org.apache.velocity.exception.ParseErrorException;
+import org.apache.velocity.exception.ResourceNotFoundException;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpHeaders;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.MediaType;
+import org.springframework.http.ResponseEntity;
+import org.springframework.stereotype.Controller;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestMethod;
+import org.springframework.web.bind.annotation.RequestParam;
+
+import edu.asu.spring.quadriga.domain.factories.IRestVelocityFactory;
+import edu.asu.spring.quadriga.domain.workbench.IProject;
+import edu.asu.spring.quadriga.exceptions.QuadrigaAccessException;
+import edu.asu.spring.quadriga.exceptions.QuadrigaStorageException;
+import edu.asu.spring.quadriga.exceptions.RestException;
+import edu.asu.spring.quadriga.service.IRestMessage;
+import edu.asu.spring.quadriga.service.network.INetworkTransformationManager;
+import edu.asu.spring.quadriga.service.network.domain.ITransformedNetwork;
+import edu.asu.spring.quadriga.service.workbench.IRetrieveProjectManager;
+import edu.asu.spring.quadriga.transform.Node;
+
+/**
+ * Controller for concept related rest api's exposed to other clients Client
+ * 
+ * @author Suraj Nilapwar
+ * 
+ */
+@Controller
+public class SearchNetworkRestController {
+
+    private static final Logger logger = LoggerFactory.getLogger(SearchNetworkRestController.class);
+
+    @Autowired
+    private IRestMessage errorMessageRest;
+
+    @Autowired
+    private IRestVelocityFactory restVelocityFactory;
+    
+    @Autowired
+    private IRetrieveProjectManager projectManager;
+    
+    @Autowired
+    private INetworkTransformationManager transformationManager;    
+    
+    /**
+     * Rest interface to search concept in list of projects
+     * http://<<URL>:<PORT>>/quadriga/rest/networks/search/{conceptId}/{projectIds}

```

Comment:
This is not correct, is it? You are expecting conceptId and projectIds as parameters not as path values.